packages/labs/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@blueprintjs/labs",
-  "version": "0.6.0",
+  "version": "0.7.0",
   "description": "Incubator for unstable and in-development Blueprint components",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "style": "dist/blueprint-labs.css",
   "unpkg": "dist/labs.bundle.js",
   "dependencies": {
-    "@blueprintjs/core": "^1.25.0",
+    "@blueprintjs/core": "^1.26.0",
```

Comment:
Nope, I think we need this. See: https://github.com/palantir/blueprint/pull/1485/files